pokerrors/errors.go

```diff
@@ -0,0 +1,128 @@
+package pokerrors
+
+import (
+	"errors"
+	"net/http"
+
+	"github.com/wmaldonadoc/academy-go-q42021/constants"
+)
+
+//	APIClientError - Represents exceptions with the following fields:
+//	- Message
+//	- HTTPStatus
+//	- Code: Represents a custom error code
+//	- Err: An Error from errors package
```

Comment:
Be consistent with all document comments; do not use tab, use a single space.

Please fix this here and in all other occurrences.
```suggestion
// APIClientError - Represents exceptions with the following fields:
// - Message
// - HTTPStatus
// - Code: Represents a custom error code
// - Err: An Error from errors package
```